client/my-sites/site-settings/theme-enhancements.jsx

```diff
@@ -195,7 +206,7 @@ class ThemeEnhancements extends Component {
 					{ this.renderToggle(
 						'wp_mobile_featured_images',
 						! minilevenModuleActive,
-						translate( 'Hide all featured images' )
+						translate( 'Show featured images' )
```

Comment:
Just double checking that the change from Hide all to Show was intentional.

Reply:
Yes! Thank you for checking. We have had this incorrectly labelled for a long time. Jetpack looks like this:

![Screenshot 2019-07-17 at 13 29 35](https://user-images.githubusercontent.com/411945/61375502-f4f88800-a896-11e9-8dce-0b08fcb8abc8.png)